src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -74,6 +74,20 @@ typedef struct WorkerAggregateWalkerContext
 	bool pullDistinctColumns;
 } WorkerAggregateWalkerContext;
 
+/*
+ * LimitOrderByReference a structure that is used commonly while
+ * processing sort and limit clauses.
+ */
+typedef struct LimitOrderByReference
```

Comment:
It reads a bit like "order by reference". Maybe swap it to `OrderByLimitReference`, since that's also the SQL order.

Reply:
Makes sense